cluster/gce/trusty/kube-manifests/kube-apiserver.manifest

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
Which volumes & volumeMounts you removed here? My concern is that we create too many duplicated configs with small treaks.